genienlp/tasks/almond/__init__.py

```diff
@@ -147,58 +144,19 @@ def _detokenize_cjk_chars(self, sentence):
                 i += 1
         return "".join(output)
 
-    def tokenize(self, sentence, field_name=None):
+    def preprocess_field(self, sentence, field_name=None):
+        if self.override_context and field_name == 'context':
```

Comment:
We were doing 'NFD' normalization in masked_tokenizers before to reduce number of OOVs in BERT. We should add it somewhere here.  
Also `tokenize_chinese_chars` and `strip_accents` for BertTokenizer need to be True unconditionally to mimic previous behavior.

Reply:
Actually, we need to pass `do_basic_tokenize=False` to BertTokenizer otherwise ThingTalk symbols get all messed up, so those flags have no effect. We'll need to do things differently.